.pylintrc37

```diff
@@ -515,7 +515,7 @@ max-locals=15
 max-parents=7
 
 # Maximum number of public methods for a class (see R0904).
-max-public-methods=20
+max-public-methods=40
```

Comment:
Guessing this is browser.py below busting the limit? This rule might be meaningless for this file or context.